deploy/clowdapp.yaml

```diff
@@ -910,7 +909,7 @@ parameters:
     value: "path"
   - name: PULP_DEFAULT_FILE_STORAGE
     description: Pulp default file storage
-    value: "storages.backends.s3boto3.S3Boto3Storage"
+    value: "storages.backends.s3.S3Storage"
   - name: PULP_MEDIA_ROOT
```

Comment:
I haven't updated the names in these `parameters` section because I'm not sure where these comes from and what it could break.
The way I understand this is that items in the `env`  sections (diff from L300:600) are the only ones referencing these names for template processing on the values. Since the keys were updated there, the envvars should have the correct names.

Let me know if I'm missing something here.